apis/nucleus/src/components/listbox/components/ListBoxSearch.jsx

```diff
@@ -75,20 +76,14 @@ export default function ListBoxSearch({
   const { translator } = useContext(InstanceContext);
   const [value, setValue] = useState('');
   const [wildcardOn, setWildcardOn] = useState(false);
-
   const inputRef = useRef();
 
   const { getStoreValue, setStoreValue } = useDataStore(model);
   const isRtl = direction === 'rtl';
 
-  const cancel = () => selections.isActive() && selections.cancel();
+  const cancel = () => selections.cancel();
 
   const abortSearch = async () => {
-    // When select is disabled we always want to allow abort (but not permitting selections).
-    const preventAbort = selectionState.selectDisabled() ? false : !selections.isModal();
-    if (preventAbort) {
-      return;
-    }
```

Comment:
For example revert this one?

Reply:
Why do you think we need preventAbort ?

Reply:
Because when user haven't initiate the search then no need to call `abortListObjectSearch`.

Reply:
with the change in #1557 i think we can keep preventAbort (then it will prevent unnecessary engine call in some cases)
if we decide to not do that then the preventAbort should be removed

Reply:
As discussed offline we decided to remove preventAbort. No action needed.